runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsOverlay.java

```diff
@@ -31,11 +31,8 @@
 import java.awt.Graphics2D;
 import java.awt.Polygon;
 import java.awt.Rectangle;
+import java.util.*;
```

Comment:
Per [Runelite code conventions ](https://github.com/runelite/runelite/wiki/Code-Conventions), wild card imports should not be used

Reply:
Sorry, Idea did this automatically, fixed now